src/apps/frontend/pages/authentication/phone-login/phone-login-form.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import {
+  Button,
+  Flex,
+  FormControl,
+  Input,
+  Select,
+  VerticalStackLayout,
+} from '../../../components';
+import COUNTRY_SELECT_OPTIONS from '../../../constants/countries';
+import routes from '../../../constants/routes';
+import { AsyncError } from '../../../types';
+import { ButtonKind, ButtonType } from '../../../types/button';
+
+import usePhoneLoginForm from './phone-login-form.hook';
+
+interface PhoneLoginFormProps {
+  onSendOTPSuccess: () => void;
+  onError: (error: AsyncError) => void;
+}
+
+const PhoneLoginForm: React.FC<PhoneLoginFormProps> = ({ onError, onSendOTPSuccess }) => {
+  const { formik, isSendOTPLoading } = usePhoneLoginForm({ onSendOTPSuccess, onError });
+
+  const setFormikFieldValue = (fieldName: string, data: string) => {
+    formik.setFieldValue(fieldName, data)
+      .then().catch((err) => { onError(err as AsyncError); });
+  };
+
+  const handleChangePhone = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    const { value } = target;
+    setFormikFieldValue('phoneNumber', value);
+  };
+
+  const handleChangeSelect = ({ target }: React.ChangeEvent<HTMLSelectElement>) => {
+    const { value } = target;
+    const [countryCode, country] = value.split(', ');
+    setFormikFieldValue('country', country);
+    setFormikFieldValue('countryCode', countryCode);
+    setFormikFieldValue('phoneNumber', '');
+  };
+
+  return (
+    <>
```

Comment:
Seems like everything is wrapped in `form` tag let's remove this empty tag if not needed.

Reply:
Removed unneccessary wrapping